Discuss Exception processing in Websocket Chain

While working with Netty, discovered that the [chain update](https://github.com/OpenLiberty/open-liberty/blob/b6cbb2bd7d8ffaa0aec4c386f73410ca71a2fe0d/dev/com.ibm.ws.wsoc/src/com/ibm/ws/wsoc/outbound/WsocChain.java#L300-L309) for Websocket has no exception processing when starting the chain. In Netty we are following the same approach but should we perhaps improve this for the future?

See comment here as well: https://github.com/OpenLiberty/open-liberty/pull/31828/files

Oddy, I also commented out the legacyUpdate code and I was still able to connect to the websocket endpoint? 

But when I ran a FAT, I started to see errors, such as:
```
[err] Caused by: com.ibm.wsspi.channelfw.exception.InvalidChainNameException: Chain configuration not found in framework, WsocOutboundHttp
```

--- a/dev/com.ibm.ws.wsoc/src/com/ibm/ws/wsoc/outbound/WsocChain.java
+++ b/dev/com.ibm.ws.wsoc/src/com/ibm/ws/wsoc/outbound/WsocChain.java
@@ -364,14 +364,13 @@ public class WsocChain {
             newConfig.validConfiguration = true;
 
         } catch (ChannelException e) {
-            // handleStartupError(e, newConfig); // FFDCIgnore: CFW will have logged and FFDCd already
+            // FFDCIgnore: CFW will have logged and FFDCd already
         } catch (ChainException e) {
-            // handleStartupError(e, newConfig); // FFDCIgnore: CFW will have logged and FFDCd already
+            // FFDCIgnore: CFW will have logged and FFDCd already
         } catch (Exception e) {
-            // The exception stack for this is all internals and does not belong in messages.log.
-            //  Question: need error message here?
-            //  Tr.error(tc, "config.httpChain.error", tcpName, e.toString());
-            //  handleStartupError(e, newConfig);
+            if(TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                Tr.debug(this, tc, "Error updating chain " + chainName, this, e);
+            }
         }
 
     }
@@ -391,9 +390,9 @@ public class WsocChain {
             else
                 owner.unsecureBootstrap = nettyBootstrap;
         } catch (NettyException e) {
-            // When updating the legacy chain, no error processing took place
-            // due to already logged exceptions from channel framework. 
-            // https://github.com/OpenLiberty/open-liberty/issues/31815
+            if(TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                Tr.debug(this, tc, "Error updating chain " + chainName, this, e);
+            }
         }
 
     }

